tests/test_schema.py

```diff
@@ -149,6 +149,29 @@ def test_schema_bad_prop_03(self):
             };
         """
 
+    @tb.must_fail(errors.InvalidReferenceError,
+                  "pointer 'test::bar' does not exist",
```

Comment:
can we use the real class name (i.e. 'link' or 'property') instead of 'pointer' here?

Reply:
OK, I've updated the message to refer tot he specific type.